composables/httpd.ts

```diff
@@ -0,0 +1,22 @@
+const PER_PAGE = 100
+
+function createHttpdBaseURL(domainName = 'seed.radicle.xyz') {
+  return `https://${domainName}/api/v1`
+}
+
+interface FetchOptions {
+  domainName?: string
+}
+
+interface UseFetchIssuesOptions extends FetchOptions {
+  rid: string
+}
+
+export async function useFetchIssues({ rid, domainName }: UseFetchIssuesOptions) {
+  return await useHttpdFetch('/projects/{rid}/issues', {
+    path: { rid },
+    baseURL: createHttpdBaseURL(domainName),
+    // @ts-expect-error - query params are not typed
+    query: { perPage: PER_PAGE },
```

Comment:
This should be set in the plugin for all requests of `useHttpdFetch()` by default with a default value of `1000`.

It's fine, httpd is usually local, and hitting the db once is anyway faster overall than with successive paginated reqs. We can optimize further if we are ever fortunate enough to have that issue.

Reply:
What about endpoints without pagination (e.g. single item GET, or POST/PATCH)? If the backend ignores the query param it's fine, but it could be an issue otherwise.

Wasn't concerned about performance, just thought 100 would be enough not to do pagination, sure 1000 is fine by me.

Reply:
>What about endpoints without pagination (e.g. single item GET, or POST/PATCH)? If the backend ignores the query param it's fine, but it could be an issue otherwise.

We'll cross that bridge when we get there.

>Wasn't concerned about performance, just thought 100 would be enough not to do pagination, sure 1000 is fine by me.

100 also enough by me. 420 too. It doesn't matter much really, it's pretty arbitrary.